src/pages/Accounts/index.tsx

```diff
@@ -113,15 +125,21 @@ const getAccountsColumns = ({ formatAddress, onShowSendFundsModal, onShowWithdra
         );
       }
       return (
-        <ActionsWrapper>
-          <Button title={'Send'} onClick={onShowSendFundsModal(accountInfo.address)} />
-          <Button
-            title={'Get'}
-            disabled={true}
-            // onClick={onShowSendFundsModal(accountInfo.address)}
-          />
-          {accountInfo.signerType === 'Local' && <Button title={'Delete'} onClick={onShowDeleteLocalAccountModal(accountInfo.address)} />}
-        </ActionsWrapper>
+        <>
+          <ActionsWrapper>
+            <Button title={'Send'} onClick={onShowSendFundsModal(accountInfo.address)} />
+            <Button
+              title={'Get'}
+              disabled={true}
+              // onClick={onShowSendFundsModal(accountInfo.address)}
```

Comment:
remove?

Reply:
Handler for this is added in the other PR